azure-pipelines/powershell-templates/powershell-execution-tests-template.yml

```
@@ -7,10 +7,22 @@ steps:
   inputs:
     targetType: 'inline'
     script: |
-        if (!(Get-Module -Name Microsoft.Graph -ListAvailable)) {
-            Install-Module Microsoft.Graph -Repository PSGallery -Scope AllUsers -Force -AllowClobber
-            Import-Module Microsoft.Graph -Global -Force 
+        $module = Get-Module -Name Microsoft.Graph -ListAvailable
+        if($module.Count -gt 0){
+          $module[0] | %{ 
+          if($_.Version -eq "1.10.0"){
+          echo "Latest version already installed"
+          }else{
+          echo "Installing latest version"
+          Install-Module Microsoft.Graph -Repository PSGallery -Scope AllUsers -Force -AllowClobber
+          Import-Module Microsoft.Graph -Global -Force  
+          }}
+        }else{
+          echo "Installing latest version"
+          Install-Module Microsoft.Graph -Repository PSGallery -Scope AllUsers -Force -AllowClobber
+          Import-Module Microsoft.Graph -Global -Force
         }
```

`%` loops over all items in a list, `module[0]` is an item in the list, so we don't need a loop there. I think we can simplify is as follows:
```suggestion
        $modules = Get-Module -Name Microsoft.Graph -ListAvailable
        if ($modules.Count -gt 0 -and $modules[0].Version -eq "1.10.0") {
          echo "Latest version already installed"
        }
        else {
          echo "Installing latest version"
          Install-Module Microsoft.Graph -Repository PSGallery -Scope AllUsers -Force -AllowClobber
          Import-Module Microsoft.Graph -Global -Force  
        }
```

---

In addition, is there a way to check the latest version dynamically?

cc: @peombwa